README.md

```diff
@@ -80,7 +80,7 @@ you may need to install packages like PIL@1.1.7 and requests if not installed
 
 * ### End To End Encryption
 It is a simple program to implement and understand the basic of end_to_end encryption.
-Here i am using caesar cipher to encrpt nbut in reality they Use algotihms lile SHA-1, RSA etc.
+Here I am using caesar cipher to encrpt but in reality they use algotihms lile SHA-1, RSA etc.

```

Comment:
Hey, @bhumijgupta Can you please change encrpt spelling to encrypt. Thanks

Reply:
@Logan1x I changed the spelling!!